Fixed ts-utils import

Dantemss opened the pull request:
For: https://github.com/openstax/unified/issues/2727

Need this because of ESM or something

Dantemss commented:
How odd... then the GH action that uses cjs fails... I'll investigate some more

jivey commented on jest.config.js:
```diff
@@ -13,7 +13,4 @@ module.exports = {
     "node"
   ],
-  moduleNameMapper: {
-    "(.*)@openstax/ts-utils(.*)": "$1@openstax/ts-utils/$2.js"
```
I thought this was doing the same thing as manually adding the extensions to the imports, maybe not?

Dantemss replied:
It was fine in the spec files but then in the regular imports that needed the extension outside of jest it started duplicating the extension for me.

jivey commented on src/components/DropdownMenu.spec.tsx:
```diff
@@ -17,70 +17,75 @@ describe('DropdownMenu', () => {
-    const component = renderer.create(<TestMenu variant='light'/>);
-    expect(component.toJSON()).toMatchSnapshot();
+  // Need to isolate tests from the 2 libraries in these blocks otherwise we get warnings about using the wrong act()
+  describe('@testing-library/react', () => {
```
Interesting, I wouldn't expect a `describe` to provide more isolation than an `it`, unless using the module name as the describe name is doing something special here (I don't think so?)

TomWoodward replied:
yea i'm also not sure how the describe is helping in this case

jivey reviewed (commented):
What lead to needing this change? I can probably learn something here regarding how bundling works if you will indulge me :)

Dantemss commented:
@jivey the change was needed because when I added the ts-utils import for the assertions like Tom asked the tests passed but the code failed to run. Then when I added the .js extension the tests failed.
